testing/hiawatha/hiawatha.initd

```diff
@@ -2,12 +2,21 @@
 
 supervisor=supervise-daemon
 
-name="Webserver"
-description="Secure and Advanced Hiawatha Webserver"
+name="Hiawatha webserver"
+description="Secure and advanced webserver"
 
 command=/usr/sbin/hiawatha
-command_args_foreground="-d"
+command_args_foreground="-d 1>/dev/null"
```

Comment:
put it on command_args. command_args_foreground is meant for the command switch that makes the program stay on the foreground